conf/messages.en

```diff
@@ -134,6 +134,12 @@ compliant.nine.p2.2=Digital Accessibility Centre
 compliant.nine.p2.3=The full service was tested by HMRC and included disabled users.
 compliant.nine.p3=This page was prepared on [date when it was first published]. It was last updated on [date when it was last updated].
 
+dob.label = Date of birth
```

Comment:
I feel the key should be prefixed with chargeG.

Reply:
it's kept generic intentionally so that it can be reused if we have another date of birth in any other charge

Reply:
OK